src/methods/utility/asDerivative.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Args,
+  BaseTxInfo,
+  createMethod,
+  OptionsWithMeta,
+  UnsignedTransaction,
+} from '../../util';
+
+export interface UtilityAsDerivativeArgs extends Args {
+  /**
+   * The sub-account index of the origin.
+   */
+  index: number | string;
+  /**
+   * The call to be dispatched from pseudonym account derived from the origin.
+   * To take advantage of txwrapper methods, this could be an array of
+   * UnsignedTransaction.method.
+   */
+  call: { callIndex?: string; args?: string } | string;
+}
+
+/**
+ * Send a call through an indexed pseudonym of the sender.
+ *
+ * Filter from origin are passed along. The call will be dispatched with an origin which
```

Comment:
"Filter from origin are passed along": I'm not sure I understand what this means. Could be that a reader with a better understanding of these calls have no problem but I could use a little more info.

Reply:
I'll look into this. For context though all these are just copy and pasted from the substrate source so perhaps worth making a PR to update the docs there.

Reply:
Lmk what you think of these updates: https://github.com/paritytech/txwrapper/pull/260/commits/ddb09535f8c1e30f2592ab879eac35eb5c601e72